### PR TITLE
Complete get_fan_object_electric_power

### PR DESCRIPTION
**Pull request by weilixu**

### CODE REVIEW

File: rct229/rulesets/ashrae9012019/ruleset_functions/get_fan_object_electric_power.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    Returns
+    -------
+    fan_elec_power: a pint quantity object with a unit of Btu/hr.
+    None if missing key parameters for calculating the fan power
+
+    """
+    fan_elec_power = None
+    fan_spec_method = getattr_(fan, "fan", "specification_method")
+    if fan_spec_method == FAN_SPECIFICATION_METHOD.SIMPLE:
+        # fan_elec_power = getattr_(fan, "fan", "design_electric_power")
```
Comment (yunjoonjung-PNNL) [inline rct229/rulesets/ashrae9012019/ruleset_functions/get_fan_object_electric_power.py:29]: Could you check if we delete this line?

**Review by yunjoonjung-PNNL — APPROVED** [review]
It looks great to me. Thanks Weili.

File: rct229/rulesets/ashrae9012019/ruleset_functions/get_fan_object_electric_power.py
Comment (charliepnnl) [inline rct229/rulesets/ashrae9012019/ruleset_functions/get_fan_object_electric_power.py:1]: The RDS for this function doesn't make physical sense to me.  To get the power of a fan would seem to need the pressure rise, the flow rate, and the motor efficiency:
```
power = (pressure rise) (flow rate) / (motor efficiency)
```

**Review by charliepnnl — COMMENTED** [review]
I am seriously questioning the RDS for this one.

**Pull request merged**